src/job_metadata.c

```diff
@@ -1475,3 +1478,54 @@ JobTableExists(void)
 
 	return jobTableOid != InvalidOid;
 }
+
+
+/*
+ * ParseSchedule attempts to parse a cron schedule or an interval in seconds.
+ * The returned pointer is allocated using malloc and should be freed by the
+ * caller.
+ */
+static entry *
+ParseSchedule(char *scheduleText)
+{
+	uint32 secondsInterval = 0;
+
+	/*
+	 * Parse as interval on seconds or fall back to trying cron schedule.
+	 */
+	if (ParseInterval(scheduleText, &secondsInterval))
+	{
+		entry *schedule = calloc(sizeof(entry), sizeof(char));
+		schedule->secondsInterval = secondsInterval;
+		return schedule;
+	}
+
+	return parse_cron_entry(scheduleText);
+}
+
+
+/*
+ * ParseInterval returns whether scheduleText is of the form
+ * <positive number> second[s].
+ */
+static bool
+ParseInterval(char *scheduleText, uint32 *secondsInterval)
+{
+	char plural = '\0';
+	char extra = '\0';
+	int numParts = sscanf(scheduleText, "%u second%c %c", secondsInterval, &plural,
```

Comment:
you could convert scheduleText to lowerCase so that even if it is given as 6 SECONDS or 6 SECOND it works as expected